release-notes/VERSION-2.x

```diff
@@ -5,6 +5,11 @@ Project: jackson-databind
 ------------------------------------------------------------------------
 
 2.19.4 (29-Oct-2025)
+
+#5393: `@JsonAnyGetter property gets included in generated schema since 2.19.0`
```

Comment:
Needs to move above as this won't be in 2.19.4 (was already released) -- Like:

```
(not yet released)

#5393: `@JsonAnyGetter` property ....
```

(no 2.19.5 since branch is officially closed -- ok to merge there but next release to contain fix will be 2.20.2 (or 2.21.0).